nodeadm/internal/featuregates/featuregates.go

```diff
@@ -0,0 +1,31 @@
+package featuregates
+
+type Feature string
```

Comment:
This type should be in the `internal/api` package, since it mirrors an external API type. The `FeatureGate` struct and its helpers can stay here

Reply:
`internal/api` has dependency on `FeatureGate`, move `Feature` to there will introduce the cycle dependency.

Reply:
Makes sense to me!